prac_05/hex_colours.py

```diff
@@ -0,0 +1,28 @@
+COLORS = {
```

Comment:
Dictionary should be named "(keywords)_to_(value)".

Reply:
I will fix that, thanks for the comment!